Orchid.Jint/JsExecutionContext.cs

```diff
@@ -147,7 +149,7 @@ public void Dispose()
 
             if (null != _engine)
             {
-                _engine.Destroy();
+ //               _engine.Dispose();
```

Comment:
should this line be commented? if not used can it be removed?

Reply:
In this version of Jint, engines are not disposable, but in the next version they are, so I left it as a reminder. However, that is pretty obscure, so I rewrote it to test for disposability.